templates/shared/searchinput.tmpl

```diff
@@ -1 +1 @@
-<input type="search" spellcheck="false" name="q" maxlength="255" placeholder="{{ctx.Locale.Tr "explore.search"}}…"{{if .Value}} value="{{.Value}}"{{end}}{{if .Disabled}} disabled{{end}}>
+<input type="search" name="q"{{if .Value}} value="{{.Value}}"{{end}}{{if .Disabled}} disabled{{end}} spellcheck="false" maxlength="255" placeholder="{{ctx.Locale.Tr "explore.search"}}…">
```

Comment:
Happy merge conflict resolution :tada:

Reply:
Happy to resolve, done in 1323919924b3b98cd54d0c047657416212af6866 & a9851be66818fc787aafee79c93cea6c11286690